### PR TITLE
Add mssql global filter option

### PR DESCRIPTION
**Pull request by karelhala**
## New MSSQL filter is available

Chrome service recently added new filter to global filter https://github.com/RedHatInsights/insights-chrome/pull/1776 so let's pass it down the foodchain. This PR is similiar to https://github.com/RedHatInsights/insights-inventory-frontend/pull/1491

### CODE REVIEW

**Review by johnsonm325 — APPROVED** [review]
Looks straight-forward

File: src/routes/InventoryTable.js [hunk 1]
```diff
@@ -148,8 +148,10 @@ const Inventory = ({
                     system_profile: {
                         ...globalFilter?.filter?.system_profile,
                         ...workloads?.SAP?.isSelected && { sap_system: true },
-                        ...workloads && workloads['Ansible Automation Platform']?.isSelected
+                        ...workloads?.['Ansible Automation Platform']?.isSelected
                             && { ansible: { controller_version: true } },
+                        ...workloads?.['Microsoft SQL']?.isSelected
+                            && { mssql: { version: 'not_nil' } },
```
Comment (johnsonm325) [inline src/routes/InventoryTable.js:154]: According to Asa, it's a good idea to update this with `&& { mssql: 'not_nil' },` to prep for the future. Currently there is only one child, but that may change and this is a good way to prepare. See this exchage: https://ansible.slack.com/archives/C02T90PLJ5V/p1646040794757729

**Review by johnsonm325 — CHANGES_REQUESTED** [review]
This change should be made after a rebase to take in Asa's change: https://github.com/RedHatInsights/insights-inventory-frontend/pull/1502

File: src/routes/InventoryTable.js (hunk 1 above)
Comment (karelhala) [reply src/routes/InventoryTable.js:154]: Is this deployed? We should update the AAP filter as well. I saw a PR in chrome to update the AAP filter, but was not sure if it was fully deployed for MSSQL as well.

Comment (johnsonm325) [reply src/routes/InventoryTable.js:154]: #1502 is merged, so I guess the answer is that the AAP filter is already updated. If I understood Asa correctly, updating mssql here would prepare us for whenever mssql got another child. Once you rebase this, the AAP filter will update though.

Let me know if I'm misunderstanding your question.

Comment (karelhala) [reply src/routes/InventoryTable.js]: Sweet! Updated the code, looks even cleaner. Tested it locally and it filters properly, so everything is smooth.

**Review by johnsonm325 — APPROVED** [review]

**Pull request merged**